slidez-core/src/main/resources/db/migration/V1.1__case_insensitive_email.sql

```diff
@@ -1,9 +1,9 @@
-create collation case_insensitive
+create collation IF NOT EXISTS case_insensitive
```

Comment:
Please, don't change existing migrations. Everyone who already applied it will see hash mismatch message